### DISCUSSION TITLE
주제 변경

### DISCUSSION THREAD

**Discussion by wisdom08** [wisdom08/dev-community#24] [Ideas]
멘토님 현재 구현 예정 중인 기능들은 이전에 뷰 단을 제외하고는 이미 만들어 본거라 주제를 좀 바꿔서 진행하고 싶은데요.

오늘 제 친구의 이야기를 듣고 만들어보고 싶은게 생겼습니다.
친구가 40~50대 은행 어플과 카카오톡 채팅 기능만 이용하고, 카카오페이나 다른 어플 이용이 익숙하지 않은 분들과 업무 특성 상 
식사를 자주 합니다. 그리고 친구가 일괄적으로 계산을 하고 1/n한 금액을 친구의 계좌로 보내주는데요. 
정말 자주 바로 바로 보내지 않는 분들이 있어 스트레스를 받는 친구 이야기를 들었습니다. 
지금까지는 메모장 어플에 일일히 적었는데 적는 걸 까먹는 경우도 많고 메모를 체계적으로 하는 친구가 아니어서 이를 관리하는게 쉽지 않다고 하더라고요. 그래서 그 친구를 좀 편하게 해주고 싶어서 관련 서비스를 만들어보고 싶습니다. 만들어가는 과정이 재미있을 거 같아요. 

우선 계획했던 주니어 개발자 커뮤니티를 완성하고, 다른 서비스를 만드는게 좋을까요? 아니면 주제를 바꾸는 것도 괜찮을까요? 

### 구상해본 기능
- 로그인을 한다.
- 로그인한 유저는 미리 같이 밥 먹는 사람들의 이름 혹은 별명을 등록한다. 
- 그리고 달력에 일자 별로 미리 등록한 인원 중 해당 날짜에 밥을 먹는 사람을 선택하고 식대 총 합을 기록한다.
  - 각각 금액이 다른 경우, 개별 입력도 가능하게 한다.   
- 이체 확인 후에 그 사람을 체크해서 식대를 이체해야하는 인원 목록에서 제외시킨다.

**Comment by jongfeel** [comment]
제 의견을 말씀 드리면

우선 현재 진행하고 있는 주니어 개발자 커뮤니티를 완성하는 걸 목표로 했으면 좋겠습니다.

첫 미팅 때 말씀 드렸지만 이걸 2주면 만들 수 있다고 하셨는데
지금 한 달이 지났는데도 소셜 로그인 기능만이 구현되어 있는 상태입니다.
제가 예상했던 기간도 3달이라고 했지만 그것도 정말 최소로 잡은 거고
처음에 저한테 보내주셨던 그 기능을 다 구현하려면 기간을 가늠할 수가 없습니다.

그래서 앞으로 한 번 정도 iteration을 더 돌고 나면 개발 속도를 조금 측정해 볼 수 있을 것 같은데요.
그 속도가 지혜님이 개발 능력치라고 보시면 좋을 것 같습니다. 게임의 스탯 같은 거죠.

저는 자기 자신을 잘 알고 작업 범위를 잘 잡은 후에
최후에 동작하는 작은 소프트웨어를 완성 시키는 사람이
개발 실력이 좋은 사람이라고 평가하고 싶습니다.

그리고 저도 그렇게 시작해서 끝을 못본 흑역사들이 제 github에 남아 있는 걸 보면
다음 부터는 그러지 말아야 겠다는 결심을 많이 하게 됩니다.

---

번외로 그 친구의 고민을 해결 할 수 있는 서비스가 있는데요.
카카오톡에 1/n 정산하기 기능을 잘 사용해 보시라고 하는게 어떨까요?
서비스를 개발하는 것 보다 그 기능을 사용하는게 비용이 덜 드는 측에 속할 것 같습니다

**Reply by wisdom08** [reply]
아하 네 알겠습니다 
그 2주는 뷰 단을 제외하고 말씀을 드린 거였고 프론트 만드는게 생각보다 오래 걸리는 것같습니다. 

말씀하신대로 저를 알아가는 시간을 일단 가져야겠네요!
감사합니다.
